introduction_to_applying_machine_learning/video_game_sales/video-game-sales-xgboost.ipynb

```diff
@@ -323,206 +287,75 @@
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "job_name = 'DEMO-videogames-xgboost-' + strftime(\"%Y-%m-%d-%H-%M-%S\", gmtime())\n",
-    "print(\"Training job\", job_name)\n",
-    "\n",
     "from sagemaker.amazon.amazon_estimator import get_image_uri\n",
+    "\n",
     "container = get_image_uri(boto3.Session().region_name, 'xgboost')\n",
     "\n",
-    "create_training_params = \\\n",
-    "{\n",
-    "    \"RoleArn\": role,\n",
-    "    \"TrainingJobName\": job_name,\n",
-    "    \"AlgorithmSpecification\": {\n",
-    "        \"TrainingImage\": container,\n",
-    "        \"TrainingInputMode\": \"File\"\n",
-    "    },\n",
-    "    \"ResourceConfig\": {\n",
-    "        \"InstanceCount\": 1,\n",
-    "        \"InstanceType\": \"ml.c4.xlarge\",\n",
-    "        \"VolumeSizeInGB\": 10\n",
-    "    },\n",
-    "    \"InputDataConfig\": [\n",
-    "        {\n",
-    "            \"ChannelName\": \"train\",\n",
-    "            \"DataSource\": {\n",
-    "                \"S3DataSource\": {\n",
-    "                    \"S3DataType\": \"S3Prefix\",\n",
-    "                    \"S3Uri\": \"s3://{}/{}/train\".format(bucket, prefix),\n",
-    "                    \"S3DataDistributionType\": \"FullyReplicated\"\n",
-    "                }\n",
-    "            },\n",
-    "            \"ContentType\": \"libsvm\",\n",
-    "            \"CompressionType\": \"None\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"ChannelName\": \"validation\",\n",
-    "            \"DataSource\": {\n",
-    "                \"S3DataSource\": {\n",
-    "                    \"S3DataType\": \"S3Prefix\",\n",
-    "                    \"S3Uri\": \"s3://{}/{}/validation\".format(bucket, prefix),\n",
-    "                    \"S3DataDistributionType\": \"FullyReplicated\"\n",
-    "                }\n",
-    "            },\n",
-    "            \"ContentType\": \"libsvm\",\n",
-    "            \"CompressionType\": \"None\"\n",
-    "        }\n",
-    "    ],\n",
-    "    \"OutputDataConfig\": {\n",
-    "        \"S3OutputPath\": \"s3://{}/{}/xgboost-video-games/output\".format(bucket, prefix)\n",
-    "    },\n",
-    "    \"HyperParameters\": {\n",
-    "        \"max_depth\":\"3\",\n",
-    "        \"eta\":\"0.1\",\n",
-    "        \"eval_metric\":\"auc\",\n",
-    "        \"scale_pos_weight\":\"2.0\",\n",
-    "        \"subsample\":\"0.5\",\n",
-    "        \"objective\":\"binary:logistic\",\n",
-    "        \"num_round\":\"100\"\n",
-    "    },\n",
-    "    \"StoppingCondition\": {\n",
-    "        \"MaxRuntimeInSeconds\": 60 * 60\n",
-    "    }\n",
-    "}"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
-   "source": [
-    "%%time\n",
+    "job_name = 'DEMO-videogames-xgboost-' + strftime(\"%Y-%m-%d-%H-%M-%S\", gmtime())\n",
```

Comment:
you could also specify `base_job_name` in the Estimator constructor and a timestamp will get appended to that

Reply:
base_job_name is used in the constructor, but I forgot to delete the manual timestamp creation.  Will change.